README.md

```diff
@@ -9,6 +9,401 @@
+
+First, let's create a `MoviesShow` component. This component will need to connect to the store in order to figure out which Movie it should render, so let's put it in our `containers` directory.
+
+```javascript
+// src/containers/MoviesShow.js
+import React from 'react';
+
+export default (props) => {
+  return(
+    <div>
+      Movies Show Component!
+    </div>
+  )
+}
+```
+
+Next, let's add a nested route in our `index.js` file.
+
+```javascript
+// src/index.js
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import {createStore} from 'redux';
+import rootReducer from './reducers'
+import { Provider } from 'react-redux';
+
+import {Router, Route, IndexRoute, browserHistory} from 'react-router';
+
+import App from './components/App'
+import MoviePage from './containers/MoviePage'
+import MoviesShow from './containers/MoviesShow'
+...
+
+ReactDOM.render(
+  (<Provider store={store} >
+    <Router history={browserHistory} >
+      <Route path="/" component={App} >
+        <Route path='/movies' component={MoviePage} >
+          <Route path="/movies/:id" component={MoviesShow} />
+        </Route>
+      </Route>
+    </Router>
+  </Provider>),
+document.getElementById('container'));
+```
+
+Great, now, let's add links in our `MoviesList` component so that we can click on different movies. To do this, we'll use the `Link` component that React Router gives us.
+
+```javascript
+// src/components/MoviesList.js
+import React from 'react';
+import {Link} from 'react-router';
+export default (props) => {
+  const movies = props.movies;
+
+  return (
+    <div>
+      <div className='col-md-4'>
+        <ul>
+          {movies.map( movie =>
+            <li key={movie.id}>
+              <Link to=`/movies/${movie.id}`>{movie.title}</Link>
+            </li>)}
+        </ul>
+      </div>
+    </div>
+  )
+}
+```
+Awesome! Refresh the page at `/movies`. Now, clicking a link changes the route, but we're not actually seeing any differnet content. What gives? The problem is, we've setup a child component, but we never actually said *where* it should render on the screen.
+
+In React, we can dynamically render child components by pulling them off of the `children` property on our components props. Let's update our `MoviesPage` so that it renders it's child components underneath the `MoviesList`
+
+```javascript
+// src/containers/MoviesPage.js
+import React, { Component } from 'react';
+...
+
+class MoviesPage extends Component {
+...
+
+  render(){
+    return(
+      <div>
+        <MoviesList movies={ this.props.movies } />
+        { this.props.children }
+      </div>)
+  }
+}
+
+...
+
+export default connect(mapStateToProps, mapDispatchToProps)(MoviePage);
+```
+
+Now, any child components provided by ReactRouter will be rendered there. Awesome! Refresh again - now we see our `MoviesShow` component displayed at our dynamic route.
```

Comment:
Maybe a little more explanation here. It's a child component because of the way the routes are nested?  Basically, I think just a little extra time/ repetition to let this all sink in before diving forward might be helpful